packages/devtools/lib/src/timeline/timeline.dart

```diff
@@ -22,7 +22,7 @@ import 'timeline_protocol.dart';
 
 // Light mode is Light Blue 50 palette and Dark mode is Blue 50 palette.
 // https://material.io/design/color/the-color-system.html#tools-for-picking-colors.
-const mainUiColorLight = Color(0xFF81D4FA); // Light Blue 50 - 200
+const mainUiColorLight = Color(0xFF9EBEF9); // Blue 200 Material Dark
```

Comment:
Did you mean to change the mainUiColorLight var? Or did you mean to change the mainUiColorDark var?

Reply:
Oops I did not want to change light at all.  Good catch.